packages/ddp-client/common/livedata_connection.js

```diff
@@ -613,6 +613,11 @@ export class Connection {
       try {
         stubOptions.stubReturnValue = DDP._CurrentMethodInvocation
           .withValue(invocation, stubInvocation);
+        if (Meteor._isPromise(stubOptions.stubReturnValue)) {
+          console.warn(
```

Comment:
It might be good to use Meteor._debug for this. Many apps and some packages track messages from Meteor._debug as errors (including Monti APM) so developers would learn if it happens in production